content/en/design/components/_index.md

```diff
@@ -74,44 +74,44 @@ H4 is the default type size, and should be used for all normal body text through
 
 H5 is a smaller body text size that we use sparingly in places where space is tight, such as timestamps in the upper right of content rows, condition card filter text, “belongs to” breadcrumbs, and targets goal labels.
 
-        {{% codepen PoZObmY %}}
+        {{< codepen PoZObmY >}}
 
 ## Buttons
 Primary buttons are used for actions, such as “Next” and “Submit”. Secondary buttons are the alternative to the primary action, such as "Previous". Text links are used for less important or less commonly used actions, such as “Cancel”.
-        {{% codepen WNrZKoz %}}
+        {{< codepen WNrZKoz >}}
 
 ## Checkboxes
 Use checkboxes to select multiple items within a form. When necessary, use a descriptive error state.
-        {{% codepen YzwrRRV %}}
+        {{< codepen YzwrRRV >}}
 
 ## Radios
 Use radio buttons to select a single item within a form. When necessary, use a descriptive error state.
-        {{% codepen RwrLqqZ %}}
+        {{< codepen RwrLqqZ >}}
 
 ## Inputs
 Use an input field as a basic text control to enable the user to type a small amount of text. The input size should reflect the amount of text a user is expected to enter. When necessary, use a descriptive error state.
-        {{% codepen KKVXrYE %}}
+        {< codepen KKVXrYE >}}
```

Comment:
@esthermmoturi, this correction accidentally removed one of the brackets.

Reply:
Thanks @abbyad for catching this. Fixed by Nicole on https://github.com/medic/cht-docs/pull/1585